src/components/ProductTours/TourCollapsible.tsx

```diff
@@ -80,6 +98,17 @@ const TourCollapsible: FC<Props> = (
     reopenTour(adminUuid);
   };
 
+  const hiddenSteps = useMemo(() => {
```

Comment:
You correctly exclude tabs here that are not enabled.
It would be great to make clear why the steps are excluded, as well.

I was also thinking it would be nice to simplify the logic a bit.
In the current implementation, you have a set of steps, then define a set of hidden steps, depending on props, cache this, and then subtract the list of hidden steps from the list of steps while you are already rendering the component.

Why not just have a `useEffect` depending on the two props that takes the original list with a `.filter` function and just removes the steps if necessary? Save it in a new state and then render that modified list.

That would create a clean separation between data operations and rendering, look super simple and clear, and avoid some extra steps.